.github/workflows/reusable-build-extensions-on-manylinux.yml

```diff
@@ -125,6 +125,11 @@ jobs:
         run: |
           # Running tests of wasm_bpf requires proper ebpf running environment
           mount -t debugfs none /sys/kernel/debug
+      - if: ${{ matrix.plugin == 'wasi_nn-piper' }}
+        name: Install dependencies
+        shell: bash
+        run: |
+          bash utils/wasi-nn/install-onnxruntime.sh
```

Comment:
Could this be put in the [Dockerfile](https://github.com/WasmEdge/WasmEdge/blob/master/utils/docker/Dockerfile.manylinux_2_28-plugins-deps)?

Reply:
p.s. Modification to dockerfiles can be a separate PR if CI needs an updated docker image.

Reply:
@hydai We discussed this before at https://github.com/WasmEdge/WasmEdge/issues/3483#issuecomment-2262504175. WDYT? The current workflow passed. Is this enough to consider the behavior verified?

Reply:
Yup. The CI are all green. It's okay to add this script into dockerfiles now.

Reply:
OK. I will move this into a separate PR. This PR will have to wait for the docker PR to be merged and the updated docker image.

Reply:
docker PR: [#3614](https://www.github.com/WasmEdge/WasmEdge/pull/3614)